applications/parachain-staking.md

```diff
@@ -0,0 +1,80 @@
+# Open Grant Proposal
+
+> This document is referenced in the terms and conditions and therefore needs to contain all the required information. Don't remove any of the mandatory parts presented in bold letters or as headlines! See the [Open Grants Program Process](https://github.com/w3f/Open-Grants-Program/blob/master/README_2.md) on how to submit a proposal.
+
+* **Project:** Moonbeam Network Parachain Staking Pallet
```

Comment:
consider renaming it pallet-dpos instead and offering it to substrate so it's not moonbeam-specific